docker/Dockerfile

```diff
@@ -14,12 +14,12 @@ FROM --platform=$build_for python:3.11.2-slim-bullseye as base
 # N.B. The refs updated automagically every release via bumpversion
 # N.B. dbt-postgres is currently found in the core codebase so a value of dbt-core@<some_version> is correct
 
-ARG dbt_core_ref=dbt-core@v1.5.0rc1
-ARG dbt_postgres_ref=dbt-core@v1.5.0rc1
-ARG dbt_redshift_ref=dbt-redshift@v1.5.0rc1
-ARG dbt_bigquery_ref=dbt-bigquery@v1.5.0rc1
-ARG dbt_snowflake_ref=dbt-snowflake@v1.5.0rc1
-ARG dbt_spark_ref=dbt-spark@v1.5.0rc1
+ARG dbt_core_ref=dbt-core@v1.6.0a1
```

Comment:
Would these tags need to exist on the individual repos prior to merging the PR?

Reply:
I think we build all of the docker images off of `dbt-core`.

Reply:
Ah, I see. Yeah, so what happens is this merges in, but docker images will fail for `1.6.0a1`. Then I merge all of the adapter version bumps which depend on this one. And then it all works again. We could split it into two PRs, but we likely won't try to build the images in between.